release notes/v0.3.0.md

```diff
@@ -0,0 +1,64 @@
+
+- Exposed preconfigured mobile device emulation settings. ([#289](https://github.com/grafana/xk6-browser/pull/289))
+
+  We maintain an internal list of popular devices with their viewport size, user agent and scaling factor, which is useful to test how a web site looks and behaves on different devices. See [this example](https://github.com/grafana/xk6-browser/blob/v0.3.0/examples/device_emulation.js) for details.
+
+
+
+## Bugs fixed
+
+- Made concealed elements clickable. ([#264](https://github.com/grafana/xk6-browser/pull/264))
+
+  Previously, interacting with an element obscured by another was failing with `wait for selector did not result in any nodes`. Now we properly detect this scenario and retry the action with different scroll positions. This affects all `Element` and `Frame` `click()`, `dblclick()`, `hover()`, `check()`, `uncheck()` and `tap()` actions.
+
+- Fixed `waitForSelector()` flakiness by retrying once on failure. ([#260](https://github.com/grafana/xk6-browser/pull/260))
+
+- Fixed data race accessing `Frame.documentHandle` while switching execution contexts. ([#263](https://github.com/grafana/xk6-browser/pull/263))
+
+- Fixed keyboard press of special keys. ([#283](https://github.com/grafana/xk6-browser/pull/283))
+
+  Previously pressing some keys like Backspace, Delete, ArrowLeft and AltGraph would panic because of a wrong CDP argument.
+
+- Fixed negative metric emission in certain rare scenarios. ([#288](https://github.com/grafana/xk6-browser/pull/288))
+
+- Fixed `Page.waitForFunction()`. ([#294](https://github.com/grafana/xk6-browser/pull/294))
+
+  This was previously broken and untested, and with the fix all polling variants (`requestAnimationFrame`, DOM mutation and interval) should work correctly. See [this example](https://github.com/grafana/xk6-browser/blob/v0.3.0/examples/waitforfunction.js) for details.
+
+- Fixed `Page.innerHTML()`, `Page.innerText()` and `Page.textContent()` panic. ([#299](https://github.com/grafana/xk6-browser/pull/299))
+
+
+## Improvements
+
+- Improved detection of non-clickable elements. ([#270](https://github.com/grafana/xk6-browser/pull/270))
+
+  We now properly detect elements obscured by another element and return an error instead of waiting. This enables the fix in [#264](https://github.com/grafana/xk6-browser/pull/264).
+
+- Cleanly close the WebSocket connection when `Browser.close()` is called. ([#268](https://github.com/grafana/xk6-browser/pull/268))
+
+- Added lifecycle event validation to `Page.waitForLoadState()`. ([#300](https://github.com/grafana/xk6-browser/pull/300))
+
+- We have started updating our [JavaScript API documentation](https://k6.io/docs/javascript-api/xk6-browser/), which we know how important it is to serve as a reference while writing your scripts. This is an ardous task that won't be possible in a couple of weeks, but rest assured we're prioritizing it in all upcoming development cycles, and will be releasing gradual changes as they're done.
```

Comment:
Should we mention and link to the [roadmap](https://github.com/grafana/xk6-browser/blob/241e40269e050624be21f94b80aca2cfed630d70/ROADMAP.md) here?

It's not really a code improvement, but neither are the docs... We don't link to it from anywhere, so people might miss it. Or should we link to it from the README instead?

Reply:
I was going to ask you the same :) 👍

Reply:
So, what's your suggestion?

Reply:
👍 = I agree, let's add it :)

Reply:
As discussed over Slack, I added it both here (efcef63) and in the README (6e49bc8).